Source/Fuse.Maps/Android/MapView.uno

```diff
@@ -180,6 +188,20 @@ namespace Fuse.Maps.Android
 					m.uid
 				);
 			}
+
+			foreach(MapPolyline p in Polylines)
```

Comment:
This doesn't really belong in UpdateMarkers. Even if you did an UpdateAnnotations to unify markers and polylines you'd still have a problem where adding markers would redraw polylines and adding a polyline would redraw markers. These are conceptually separate things.